prime-router/src/main/kotlin/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -0,0 +1,54 @@
+package gov.cdc.prime.router.azure
+
+import gov.cdc.prime.router.azure.db.Tables.ACTION
+import gov.cdc.prime.router.azure.db.enums.TaskAction
+import org.jooq.impl.DSL
+import java.time.OffsetDateTime
+
+interface SubmissionAccess {
+    fun <T> fetchActions(
+        sendingOrg: String,
+        orderAscending: Boolean = false,
+        resultsAfterDate: OffsetDateTime? = null,
+        limit: Int = 10,
+        klass: Class<T>
+    ): List<T>
```

Comment:
Generics are cool and all...why use one here?  Will we be returning the data from this into different data structures?

Reply:
Trying to decide if I have a good answer for this.. mostly I was punting the decision on datatype, I knew the query but didn't know what data structure I wanted to use. I think this is reason enough to leave it as is, particularly because kotlin makes the usage of this kind of generic function so easy.